config/default/system.performance.yml

```diff
@@ -0,0 +1,17 @@
+cache:
+  page:
+    max_age: 0
```

Comment:
No caching by default? Perhaps yes.

Reply:
I think it gets overridden by settings files at runtime on acquia.